Source/Letterbook.Core/Models/Mappers/ProfileMappings.cs

```diff
@@ -9,16 +12,40 @@ public class ProfileMappings : AutoMapper.Profile
 	public ProfileMappings()
 	{
 		CreateMap<Models.Profile, FullProfileDto>(MemberList.Destination)
+			.ForMember(dto => dto.Followers, opt => opt.MapFrom(src => src.FollowersCount))
+			.ForMember(dto => dto.Following, opt => opt.MapFrom(src => src.FollowingCount))
 			.ForMember(dto => dto.Created, opt => opt.Ignore());
 		CreateMap<FullProfileDto, Models.Profile>(MemberList.Source)
 			.ForSourceMember(dto => dto.Updated, opt => opt.DoNotValidate())
 			.ForSourceMember(dto => dto.Created, opt => opt.DoNotValidate())
 			.ForSourceMember(dto => dto.Keys, opt => opt.DoNotValidate())
+			.ForSourceMember(dto => dto.Followers, opt => opt.DoNotValidate())
+			.ForSourceMember(dto => dto.Following, opt => opt.DoNotValidate())
+			.ForMember(profile => profile.Followers, opt => opt.Ignore())
+			.ForMember(profile => profile.Following, opt => opt.Ignore())
 			.ForMember(profile => profile.Updated, opt => opt.Ignore())
 			.ForMember(profile => profile.Keys, opt => opt.Ignore());
 		CreateMap<Models.Profile, MiniProfileDto>(MemberList.Destination);
 
-		CreateMap<Models.SigningKey, PublicKeyDto>()
-			.ForMember(dto => dto.PublicKeyPem, opt => opt.MapFrom<PublicKeyPemConverter>());
+		CreateMap<SigningKey, PublicKeyDto>()
+			.ConvertUsing(src => new PublicKeyDto
+			{
+				Label = src.Label,
+				Family = src.Family.ToString(),
+				PublicKeyPem = PemStringBuilder(src.PublicKey).ToString().Trim(),
+				Created = src.Created,
+				Expires = src.Expires,
+				FediId = src.FediId
+			});
```

Comment:
Replaced the value converter with a convert expression, to support projections

Reply:
Note, this is tested here https://github.com/Letterbook/Letterbook/blob/1c9250d183faf30f30a1954d3bc6f74758888dde/Tests/Letterbook.Core.Tests/MapperTests.cs#L291

The produced PEM string is byte-for-byte equal to the SSC methods.